pages/integrations/artik.md

```diff
@@ -0,0 +1,214 @@
+
+### Python
+
+Here are a few notes using the [Python SDK](https://github.com/artikcloud/artikcloud-python) with resin.io devices. Using [Dockerfile templates](/deployment/docker-templates/), start from the resin default Python images, for example:
+
+```
+FROM resin/%%RESIN_MACHINE_NAME%%-python:2.7]
+```
+
+Add the `artikcloud` dependency in your `requirements.txt` file:
+
+```
+# Uncomment to use latest released version
+artikcloud
+# Uncomment to use latest development version from GitHub
+#-e git+https://github.com/artikcloud/artikcloud-python.git#egg=artikcloud
+```
+
+Later in your `Dockerfile.template` you can then install it as:
+
+```
+COPY requirements.txt ./
+RUN pip install -r ./requirements.txt
+```
+
+Then in your application you can access the environmental variables through `os.getenv()`, and send messages through the SDK. A very simple example is as follows:
+
+```python
+import os
+import time
+import artikcloud
+
+DEVICE_ID = os.getenv('ARTIKCLOUD_DEVICE_ID')
+DEVICE_TOKEN = os.getenv('ARTIKCLOUD_DEVICE_TOKEN')
+
+# Setting up ARTIK Cloud connection
+api_client = artikcloud.ApiClient()
+api_client.set_default_header(header_name="Authorization", header_value="Bearer {}".format(DEVICE_TOKEN))
+
+# Setting up messaging
+messages_api = artikcloud.MessagesApi(api_client)
+
+# Send a new message
+message = artikcloud.MessageAction()
+message.type = "message"
+message.sdid = "{}".format(DEVICE_ID)
+message.ts = int(round(time.time() * 1000))  # timestamp, required
+message.data = {'Temperature': 25.4}
+response = messages_api.send_message_action(message)
+print(response)
+```
+
+### Node.js
+
+Here are a few notes using the [Javascript/Node.js SDK](https://github.com/artikcloud/artikcloud-js) with resin.io devices. Using [Dockerfile templates](/deployment/docker-templates/), start from the resin default Node.js images, for example:
+
+```
+FROM resin/%%RESIN_MACHINE_NAME%%-node:latest]
+```
+
+Add the `artikcloud-js` dependency in your `package.json` in your application's folder:
+
+```
+npm install artikcloud-js --save
+```
+
+Later in your `Dockerfile.template` you can then install it as:
+
+```
+COPY package.json ./
+RUN JOBS=MAX npm i --unsafe-perm --production && npm cache clean
+```
+
+Then in your application you can access the environmental variables through `process.env.VARIABLE`, and send messages through the SDK. A very simple example is as follows:
+
+__NOTE__: this code exmample is unfinished
```

Comment:
Merged to be able to preview within the team, as this is not a blocking change. Added finished example before production #268 

Also, I generally would not consider WIP sections as release-blocking on our docs, but rather communicate early, communicate often. Might need to create some kind of better signaling (e.g. more fields  besides `__NOTE:__`) of different messages regarding the docs. But so far so good, imho.